### DISCUSSION TITLE
mappings - open new tab in focused tab - mimicing the "home button"

### DISCUSSION THREAD

**Discussion by tomo90** [Jelmerro/Vieb#504] [Q&A] [answered]
Hello, I'm trying to come up with a custom mapping that allows me to open the new tab (favoritepages, topsites) in currently focused tab by pressing g+h key combination on keyboard but not opening a new tab. For example like pressing the home button in Firefox which takes me to my shortcuts in the same tab. Unfortunatelly I cannot figure it out. Is it even possible?

something like **nmap gh <???>**

**Comment by tomo90** [comment, marked as answer]
I spent almost two hours trying to figure it out. I should have given it five more minutes. LOL

The scrollpos command showed me the way I guess accidentally. Calling the command it showed me this kind of syntax for a new tab: `home 0 vieb://newtab` 

As I found afterwards searching through Vieb's help page, it's called the "Special pages", if anyone's interested.

Then it was quite easy: `map gh <:open vieb://newtab>`

YAY! 💪

**Comment by Jelmerro** [comment]
Glad you found a solution! :+1:
